system/mp/mpcomm.cpp

```diff
@@ -756,8 +760,7 @@ protected: friend class CMPPacketReader;
                 // loop with short wait time and release CS to allow other side to proceed
                 StringBuffer epStr;
                 unsigned startMs = msTick();
-
-                unsigned loopCnt = ((CONNECT_READ_TIMEOUT / retrycount) / CONNECT_TIMEOUT_INTERVAL) + 1;
+                unsigned loopCnt = CONNECT_TIMEOUT_LOOPCNT;
```

Comment:
This makes the overall timeout for the connection a lot more that it was before.
Should we make it just based on the original value of retrycount.